app/store/env/go.js

```diff
@@ -2236,13 +2236,15 @@ YUI.add('juju-env-go', function(Y) {
 
       @method getChangeSet
       @param {String} bundleYAML The bundle YAML file contents.
+      @param {String} changesToken The token identifying a bundle change set
```

Comment:
then I'm at a loss how we can provide the yaml file (say a local bundle) and use quickstart to pass it into the changeset and have it become uncomitted?